### PR TITLE
Music

### PR DESCRIPTION
**Pull request by navneet851**
# Pull Request Template

## Description

Music Stream Neptune is a music streaming application developed in Kotlin. The application allows users to play, pause, shuffle, and repeat songs. It also provides a feature to like songs and maintain a list of liked songs. Using Firebase for database and cloud storage for music storing, UI build using jetpack compose framework.

## Screenshots:

<p align="center">
  
  
  
</p>

## Additional Notes:

Open Source

### CODE REVIEW

**Comment by navneet851** [comment]
added my music streaming app to showcase my project , and also edited playstore link column to download link, so anyone provide there app link , who had not publish it to playstore.

**Comment by anandwana001** [comment]
@navneet851 please follow the Pull Request Template 

Also, don't add drive link, github link is sufficient.

**Comment by navneet851** [comment]
@anandwana001 i made changes sir. will you please confirm.

File: README.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ In this repository, you'll find an eclectic collection of projects created by ou
...
-
+| Project Name | Author                   | Description                                                                                  | Technologies Used                                                                  | Repo Link | PlayStore Link                                                                                |
+|--------------|--------------------------|----------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|-----------|-----------------------------------------------------------------------------------------------|
+| KeyPass      | Yogesh Choudhary Paliyal | Open Source Project & An Offline Password Manager. Store, manage, and take control securely. | Jetpack Compose, Hilt, Material 3, MVI (Redux), Room, Jetpack Data Store           | [GitHub](https://github.com/yogeshpaliyal/KeyPass) | [PlayStore](https://play.google.com/store/apps/details?id=com.yogeshpaliyal.keypass)          |
```
Comment (anandwana001) [inline README.md]: Why these changes?

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (navneet851) [reply README.md]: automatically spacing by editor, i resolved it

**Review by anandwana001 — APPROVED** [review]
